src/datasource.js

```diff
@@ -775,6 +767,28 @@ ngeo.DataSource = class {
   // ===================================
 
   /**
+   * @return {!ngeox.DimensionsActive} Active dimensions
+   * @export
+   */
+  get activeDimensions() {
+    const active = {};
+    const global = this.globalDimensions_ || {};
+    const inner = this.innerDimensions || {};
```

Comment:
Here you mix two things:
- the `globalDimensions` objet wich are the current values for the dimensions, from the application;
- the `innerDimensions` object which is the configuration of which dimensions to use.

It is confusing. I think `innerDimensions` should be renamed to `innerDimensionsConfig` or simply `dimensionsConfig`.
Idem for the type: `ngeox.Dimensions` could be `ngeox.DimensionsConfig`.